automated-comments: add initial approve comments for java/two-fer (Fixes #1115) (#1139)
exercism/website-copy: automated-comments/java/two-fer/use_ternary_operator.md

This solution uses an if statement, but all of the conditional logic is small

> enough that we can better express it inside a ternary expression.

Reviewer: Maybe include a link to a page explaining what a ternary expression is? That concept might not be known to everyone.

Author: Added one to https://www.baeldung.com/java-ternary-operator (looked like a pretty decent tutorial on the ternary operator.

This solution uses an if statement, but all of the conditional logic is small

> enough that we can better express it using

Reviewer 2: Perhaps:
```suggestion
enough that it can better be expressed using
```

> This solution uses an if statement, but all of the conditional logic is small

Reviewer 3: ```suggestion
This solution uses an if statement, but the conditional logic is small
```

> enough that it can better be expressed using

Reviewer 3: ```diff
- enough that it can better be expressed using
+ enough that it may be expressed using
```

Reviewer 2: ```suggestion
enough that it may be expressed using
```
